packages/kbn-test/package.json

```diff
@@ -5,16 +5,22 @@
   "license": "Apache-2.0",
   "main": "./target/index.js",
   "scripts": {
-    "build": "../../node_modules/.bin/babel src --out-dir target --delete-dir-on-start --extensions .ts,.js,.tsx --ignore *.test.js,**/__tests__/** --source-maps=inline",
-    "kbn:bootstrap": "yarn build",
-    "kbn:watch": "yarn build --watch"
+    "build": "node scripts/build",
+    "kbn:bootstrap": "node scripts/build --source-maps",
+    "kbn:watch": "node scripts/build --watch --source-maps"
   },
   "kibana": {
     "devOnly": true
   },
+  "dependencies": {
+    "@kbn/es": "link:../kbn-es",
+    "@kbn/i18n": "link:../kbn-i18n",
+    "@kbn/optimizer": "link:../kbn-optimizer"
+  },
   "devDependencies": {
     "@kbn/babel-preset": "link:../kbn-babel-preset",
     "@kbn/dev-utils": "link:../kbn-dev-utils",
+    "@kbn/expect": "link:../kbn-expect",
     "@kbn/utils": "link:../kbn-utils"
   }
-}
```

Comment:
Please install the editorconfig plugin in your editor so that this doesn't keep getting flipped back in forth in every PR

Reply:
I don't believe we have come to a consensus on if we should have newlines or not?

Reply:
I swapped it because the few I checked had trailing newlines, and I hate seeing the red notice in the Github UI. However, looks like we mostly don't use newlines so I will revert this one.

`find packages -name "package.json" | xargs tail -n 1`

Reply:
Yeah, it's not exactly enforced but the contributing guides says to install the editorconfig plugin and that config has a rule for package.json specifically (also npm/yarn don't add the extra newline IIRC) https://github.com/elastic/kibana/blob/master/.editorconfig#L11-L14

Reply:
Apparently there is an editorconfig plugin for eslint, so if we wanted to pass those files through eslint too we could, but that's a separate issue.